src/module/addandremove.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
```

Comment:
- [ ] [optional] It's not good practice to disable linters for the entire file, kindly consider fixing it instead of disabling the linter check.

Reply:
@danmainah u said my project is completed . but i cannot submit on my dashboard

Reply:
@danmainah i am waiting for ur reply kindly reply me as soon as possible

Reply:
@SabaAhmad786 sorry for the late reply, I mistakenly marked it as review instead of completed, already informed the student success,in few hours they will update the status, my apologies for the inconvinience